In this commit, we initiate our project by creating the three main cl…

eid-sonkrot opened the pull request:
…asses: 
**1.UserInterface
2. Product 
3.Inventory**
These classes form the foundation of our inventory management system and will play crucial roles in providing a user-friendly interface, handling product information, and managing overall inventory quantities.

Commit:
In this commit, we initiate our project by creating the three main classes: UserInterface, Product, and Inventory. These classes form the foundation of our inventory management system and will play crucial roles in providing a user-friendly interface, handling product information, and managing overall inventory quantities.

Alatrash814 commented on Inventory Management System/Inventory.cs:
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Delete the unused imports.

Alatrash814 commented on Inventory Management System/Product.cs:
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Delete the unused imports.

Alatrash814 commented on Inventory Management System/Product.cs:
```diff
@@ -0,0 +1,12 @@
+
+namespace Inventory_Management_System
+{
+    internal class Product
```
don't use internal here.

Alatrash814 commented on Inventory Management System/Inventory.cs:
```diff
@@ -0,0 +1,12 @@
+
+namespace Inventory_Management_System
+{
+    internal class Inventory
```
don't use internal here.

Alatrash814 reviewed (changes requested):
Please use Pascal Case in Files and Dirs Names.

Commit:
Refactor code style and filenames to PascalCase

- Updated code style to follow PascalCase naming convention for variables, functions, and classes.
- Renamed files and directories to PascalCase to improve consistency.
- Removed unnecessary library calls to optimize code and reduce dependencies.

Commit:
Refactor code style and filenames to PascalCase

- Updated code style to follow PascalCase naming convention for variables, functions, and classes.
- Renamed files and directories to PascalCase to improve consistency.
- Removed unnecessary library calls to optimize code and reduce dependencies.
-Change internale class to public

Commit:
Refactor code style and filenames to PascalCase

- Updated code style to follow PascalCase naming convention for variables, functions, and classes.
- Renamed files and directories to PascalCase to improve consistency.
- Removed unnecessary library calls to optimize code and reduce dependencies.
-Change classs from inernal to public

Commit:
[Refactor code style]

Commit:
[Refactor style code]

eid-sonkrot committed:
Refactor code style
